fetch/fetch.go

```diff
@@ -151,9 +240,31 @@ func calcUnion(paramNames []string, params []*ssm.Parameter) []*ssm.Parameter {
 		}
 	}
 
+	f.debugf("Union of describeParameters and getParametersByPath: %v\n", union)
+
 	return union
 }
 
+func (f *Fetcher) debugf(format string, a ...interface{}) {
+	if f.debug {
+		fmt.Printf("DEBUG -- %s", fmt.Sprintf(format, a...))
```

Comment:
```suggestion
		fmt.Printf("DEBUG -- " + format, a...)
```

though I think it makes more sense to use an actual logger, and then the debug flag isn't needed to be passed all the way down